client/tx/factory.go

```diff
@@ -235,6 +256,22 @@ func (f Factory) BuildUnsignedTx(msgs ...sdk.Msg) (client.TxBuilder, error) {
 	return tx, nil
 }
 
+func appendTips(clientCtx client.Context, tb client.TxBuilder, f Factory) client.TxBuilder {
```

Comment:
why do we need this? I think we can remove all instances of appendTips and their usage, no?